ome/omero-figure
# code layout

This moves all the source code js and templates files out of the Django app static directory
into a new top-level `src` directory.

The grunt `jst` and `concat` tasks compile templates and concatenate js files into `omero_figure/static/figure/templates.js` and `omero_figure/static/figure/figure.js`.

All subsequent PRs will need to be on top of this one to avoid conflicts.
To test:
- `$ grunt watch`
- Edit a js file under `src/js`, Save -> Grunt should concat -> figure.js
- Edit a template under `src/templates`, Save -> Grunt will compilte -> templates.js

NB: removed the build date that is updated with every concat of figure.js to reduce conflicts between PRs.

cc @bramalingam Best test of this PR is to begin a new feature on top of it.
